ts/session/apis/open_group_api/sogsv3/sogsV3BanUnban.ts

```diff
@@ -31,7 +32,8 @@ export const sogsV3BanUser = async (
     new Set([roomInfos.roomId]),
     new AbortController().signal,
     sequence,
-    'sequence'
+    'sequence',
+    10 * DURATION.SECONDS
```

Comment:
those calls could already timeout, so they should have one yes.
This is a higher level timeout.
This is quiet a pain, but essentially,
when we do a request, we have a timeout on that request.

Let's say you want to upload an attachment. You should allow longer for the request to get through than for a retrieving of messages for instance. Well, this is not that timeout.

The timeout used as part of `timeoutWithAbort` is a more global one.
For instance, retrieving messages should take less than 10s, so you allow 10s for the request to go through. But what if you need to rebuild a path, test a guard node, fetch from the seed node because you don't have any? The timeoutWithAbort is a timeout that will abort the request if any of the step took too long.

Reply:
Ah I see. That is quite nuanced but I get your point🙏

Reply:
Resolving